src/transformers.jl

```diff
@@ -96,10 +96,14 @@ function apply(table, t::Transform; cols=nothing, kwargs...)
 
     cnames = cols === nothing ? propertynames(columntable) : cols
 
-    return [
-        _apply(getproperty(columntable, cname), t; name=cname, kwargs...)
-        for cname in cnames
-    ]
+    if cnames isa Union{Tuple, AbstractArray}
```

Comment:
can we not use `_to_vec` here as well?

Reply:
Hmm, I think it's different to the `_to_vec` for cols.

Here: If the value is a multi-element array, return that. Else (if it's a one-element array), return the element.

Whereas for cols: if the value is _any_ array, return that. Else (if it's not an array), return it as an array.

Reply:
Incidentally, this reminds me to update the docstring with :a vs [:a] behaviour.